testnets/nobletestnet/assetlist.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "nobletestnet",
+  "assets": [
+    {
+      "description": "The controlled staking asset for Noble Chain",
+      "denom_units": [
+        {
+          "denom": "ustake",
+          "exponent": 0,
+          "aliases": [
+            "microstake"
+          ]
+        },
+        {
+          "denom": "stake",
+          "exponent": 6
+        }
+      ],
+      "base": "ustake",
+      "name": "Stake",
+      "display": "stake",
+      "symbol": "STAKE"
+    },
+    {
+      "description": "Love is a test tokenfactory asset controlled by the Strangelove Team",
+      "denom_units": [
+        {
+          "denom": "ulove",
+          "exponent": 0,
+          "aliases": [
+            "microlove"
+          ]
+        },
+        {
+          "denom": "love",
+          "exponent": 6
+        }
+      ],
+      "base": "ulove",
+      "display": "Love",
+      "name": "love",
```

Comment:
Please swap the name and display values here.

Reply:
Done, thank you!